packages/react-microlink/src/components/Card/CardContent.js

```diff
@@ -15,36 +16,46 @@ export const Content = styled.div`
 const Title = styled.h2`
   font-size: 16px;
   margin: 0 0 8px;
-  white-space: nowrap;
-  overflow: hidden;
   text-overflow: ellipsis;
+  overflow: hidden;
   max-width: 95%;
+
+  ${media.mobile`
+    white-space: nowrap;
+  `}
 `
 
 const Description = styled.p`
   font-size: 14px;
   margin: 0;
   line-height: 18px;
-  height: 54px;
-  overflow: hidden;
   text-overflow: ellipsis;
+  overflow: hidden;
   -webkit-line-clamp: 3;
   -webkit-box-orient: vertical;
+
+  ${({cardSize}) => cardSize !== 'large' && media.mobile`
+    white-space: nowrap;
+  `}
+
+  ${media.desktop`
+    height: 54px;
+  `}
 `
 
 const Url = styled.span`
   font-size: 12px;
-  white-space: nowrap;
```

Comment:
Similar to `Title`, without these styles the link could possibly overflow out of the card @Kikobeats

Reply:
I think that in this case not problem because url domains tends to be small